pagination/gob.go

```diff
@@ -18,7 +19,8 @@ func gobEncode(v interface{}) string {
 	return b.String()
 }
 
-func gobDecode(s string, v interface{}) error {
+// GobDecodePageToken decodes a gob-encoded page token into an arbitrary struct.
+func GobDecodePageToken(s string, v interface{}) error {
```

Comment:
Should we call it something more descriptive, for example StructPageToken.

Reply:
Good idea!

Reply:
Updated

Reply:
Remove Gob as well? Or too much?

Reply:
Hmm. Good point!

Reply:
Updated, and merging.